src/Dialogs/Plane/PlaneDetailsDialog.cpp

```diff
@@ -185,6 +185,7 @@ PlaneEditWidget::PolarButtonClicked() noexcept
     LoadValue(TYPE, plane.polar_name.c_str());
 
   /* reload attributes that may have been modified */
+  LoadValue(HANDICAP, static_cast<int>(plane.handicap));
```

Comment:
Let's not do this cast, this is fragile code clutter. Instead, `RowFormWidget()` should have an (inline) overload for `unsigned` which does this cast.

Reply:
agreed

Reply:
I've added the overload, you can now amend this commit to remove the cast.

Reply:
Thx, did so.